src/commands/registries/dockerHub/openDockerHubInBrowser.ts

```diff
@@ -22,7 +23,8 @@ export async function openDockerHubInBrowser(context: IActionContext, node?: Doc
     } else if (node instanceof DockerHubRepositoryTreeItem) {
         url += `r/${node.parent.namespace}/${node.repoName}`;
     } else {
-        url += `r/${node.parent.parent.namespace}/${node.parent.repoName}/tags`
+        const repoTI = <DockerHubRepositoryTreeItem>node.parent;
+        url += `r/${repoTI.parent.namespace}/${repoTI.repoName}/tags`;
```

Comment:
Do you know why it's using += here?  It doesn't look like there was anything assigned to `dockerHubUrl` previously anyway

Reply:
dockerHubUrl is defined in constants.ts